test/fixtures/env.js

```diff
@@ -1,11 +1,6 @@
 "use strict";
 
-global.log = {
-	error: () => console.error.apply(console, arguments), // eslint-disable-line no-console
-	warn: () => {},
-	info: () => {},
-	debug: () => {},
-};
+global.log = require("../../src/log.js");
```

Comment:
I think it's wiser to do that.
In fact, while running this change on some code/test I am writing right now, it showed me that I was generating a unhandled warning in one of my tests and I completely missed it because it was being silenced.